types/expressions.d.ts

```diff
@@ -2381,9 +2383,20 @@ declare module 'mongoose' {
         lang: 'js'
       };
     }
+
+    export interface Exists {
+      /**
+        * When <boolean> is true, $exists matches the documents that contain the field, including documents where the field value is null.
+        * If <boolean> is false, the query returns only the documents that do not contain the field.
+        *
+        * @see https://www.mongodb.com/docs/manual/reference/operator/query/exists/
+        */
+      $exists: boolean;
+    }
   }
 
-  type Path = string;
+  type Path = string | ObjectIdLike;
```

Comment:
No. Path is just a Path. Not ObjectIdLike.

```suggestion
  type Path = string;
```

Reply:
I created a separate type, so that one day if we have good way to generate valid paths by mongoose Schema we would already have it properly encapsulated it.